source/community/index.html

```diff
@@ -42,7 +42,7 @@ <h2>How to get involved</h2>
             <h2>Community groups & committees</h2>
             <div>There are a number of <a href="groups/">community and technical groups</a>, open to participation from all interested parties, that focus on different areas of interest related to IIIF.</div>
             <br/>
-            <div>There are also a number of standing committees made up of community and consortium members with various rules on memebrship. These include the <a href="{{ site.url }}{{ site.baseurl }}/community/trc/">Technical Review Committee</a>, <a href="{{ page.webprefix }}/api/annex/notes/editors/#editorial-team-membership-and-selection">Editorial Committee</a> and <a href="{{ site.url }}{{ site.baseurl }}/community/consortium/mou/exhibit_b/#coordinating-committee">the Coordinating Committee</a>.</div>
+            <div>There are also a number of standing committees made up of community and consortium members with various rules on memebrship. These include the <a href="/community/trc/">Technical Review Committee</a>, <a href="/api/annex/notes/editors/#editorial-team-membership-and-selection">Editorial Committee</a> and <a href="/community/consortium/mou/exhibit_b/#coordinating-committee">the Coordinating Committee</a>.</div>
```

Comment:
With links from the website to the api, you need to use `{{ page.webprefix }}` otherwise it will fail travis because the api side isn't present.